javascript/javascript2/week3/gamepresser.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="game.css" />
+    <title>fastest presser</title>
+</head>
+<body>
+    <header>
+        <h2>Fastest Presser</h2>
+    </header>
+
+    <div class="inputtag"><input type="number" id="number" name="game" placeholder="Set Seconds" min="0"><br><br></div>
+   
+    <div id="startbutton"><button id="button">Start game!</button></div>
+    <div id="time">
+    <P id="count_down"></P>
```

Comment:
```suggestion
    <p id="count-down"></P>
```
Never use uppercase for tags, classes or id's